code/modules/projectiles/projectile.dm

```diff
@@ -43,6 +43,7 @@
 	var/taser_effect = FALSE //If set then the projectile will apply it's agony damage using stun_effect_act() to mobs it hits, and other damage will be ignored
 	var/check_armour = ARMOR_BULLET //Defines what armor to use when it hits things. Full list could be found at defines\damage_organs.dm
 	var/projectile_type = /obj/item/projectile
+	var/shrapnel_type = /obj/item/material/shard/shrapnel
```

Comment:
shrapnel type is intentionally null- if it's null, it uses shard behavior on embed, if it's non-null, it has special behaviour

Reply:
What are you on about ?!

This is so that we have other kind of shrapnel than steel shrapnel. And **everything** shrapnel-related is handled on the carbon mob. It was the only way for me to allow the projectile to embed as shrapnel without it magically turning into steel shrapnel.
Beside, I looked at the code, the shrapnel was previously _**hardcoded**_ to be steel shrapnel, and no exceptions were allowed.

Reply:
He answered on discord, he forgot his shrapnel changes weren't in yet.